packages/react-link/tsconfig.json

```diff
@@ -14,7 +14,8 @@
     "strict": true,
     "moduleResolution": "node",
     "preserveConstEnums": true,
-    "lib": ["es5", "dom"],
+    "skipLibCheck": true,
+    "lib": ["ES2015", "DOM"],
```

Comment:
What's the reason for adding es2015 lib? Previously we'd limited to es5 due to lack of confirmation that we're able to drop IE 11 for converged. (If that has changed, switching to es2015 is fine.)

Reply:
Last time I checked we will not target IE11

Reply:
The reason is react-tabster, that apparently depends on es2015 lib